gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformer.scala

```diff
@@ -145,9 +160,9 @@ case class WholeStageTransformer(child: SparkPlan, materializeInput: Boolean = f
       maxFields,
       printNodeId = printNodeId,
       indent)
-    if (verbose && planJson.nonEmpty) {
+    if (verbose && wholeStageTransformerContext.isDefined) {
```

Comment:
Can we still keep the previous logic by just checking whether it is empty?

Reply:
It is same with the previous logic. I just change the type from `String` to `Option[String]`